Packs/Legacy/Integrations/integration-MISP.yml

```diff
@@ -666,6 +666,8 @@ script:
       description: Hash of the file to query. Supports MD5, SHA1 and SHA256. Notice         that
         you can pass comma-separated multiple values to efficiently retrieve multiple
         responses.
+      default: true
+      isArray: true
```

Comment:
need to check if `isArray` is actually supported in the code

Reply:
reverting